lintrules/ansible/NonBuiltinAction.py

```diff
@@ -0,0 +1,33 @@
+"""
+This checks whether an action being used is not a builtin action. This has
+become far easier to check in newer versions of Ansible.
+"""
+
+from typing import Any, Dict, Union, Optional
+
+from ansiblelint.rules import AnsibleLintRule
+from ansiblelint.file_utils import Lintable
+
+# Theoretically, we should allow this to expand to other namespaces that we
+# trust to be present in the target environment. Who knows what Ubuntu 22.04
+# or newer Mint versions may include!
+ALLOWED_NAMESPACES = ["ansible.builtin"]
```

Comment:
Luckily, everything we use right now is in `ansible.builtin`/`ansible-core` as far as I can tell by running this new lint rule.